css/index.css

```diff
@@ -1,146 +1,402 @@
+    }
+    
+    .middle-content .boxes .box5 {
+        width: 12.5%;
+        height: 100px;
+        background: crimson;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    
+    .middle-content .boxes .box6 {
+        width: 12.5%;
+        height: 100px;
+        background: forestgreen;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    
+    .middle-content .boxes .box7 {
+        width: 12.5%;
+        height: 100px;
+        background: darkorchid;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    
+    .middle-content .boxes .box8 {
+        width: 12.5%;
+        height: 100px;
+        background: hotpink;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    
+    .middle-content .boxes .box9 {
+        width: 12.5%;
+        height: 100px;
+        background: indigo;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    
+    .middle-content .boxes .box10 {
+        width: 12.5%;
+        height: 100px;
+        background: dodgerblue;
+        margin: 20px 2.5%;
+        color: white;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    /* End boxes Color */
+    /* Start About Page work */
+    
+    .aboutsection {
+        border-bottom: 1px dashed black;
+    }
+    /* about section paragraph */
+    
+    .aboutsection p {
+        padding: 20px;
+    }
+    
+    .boxmain {
+        display: flex;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+    
+    .leftbox,
+    .rightbox {
+        border-bottom: 1px dashed black;
+    }
+    
+    button {
+        margin: 4%;
+        border-radius: 25px solid black;
+    }
+    /* work together section paragraph */
+    
+    .worktogethersection p {
+        padding: 20px;
+    }
+    /* End About Page work */
+    
+    .top-content {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        margin-bottom: 20px;
+        border-bottom: 1px dashed black;
+    }
+    
+    .top-content .text-container {
+        width: 48%;
+        padding: 0 1%;
+        padding-bottom: 20px;
+    }
+    
+    .middle-content {
+        margin-bottom: 20px;
+        border-bottom: 1px dashed black;
+    }
+    
+    .middle-content h2 {
+        padding: 0 2%;
+        margin-bottom: 0;
+    }
+    
+    .middle-content .boxes {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+    }
+    
+    .middle-content .boxes .box {
+        width: 12.5%;
```

Comment:
love that you used % but if you are going to set width to % set height to % as well!